src/main.rs

```diff
@@ -52,13 +59,13 @@ pub fn main() {
     // Print the version, if requested:
     if options.flags.version {
         println!("ASIMOV {}", env!("CARGO_PKG_VERSION"));
-        exit(EX_OK);
+        return Ok(EX_OK);
```

Comment:
Just to note: unlike the error return, it is fine to change this to a `Ok()` return. `exit()` was used here just for uniformity between error returns and normal returns.